eks.clusters.cnp.example.org/composition.yaml

```diff
@@ -0,0 +1,414 @@
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: Composition
+metadata:
+  name: eks.clusters.cnp.example.org
+  labels:
+    provider: aws
+spec:
+  writeConnectionSecretsToNamespace: crossplane-system
+  reclaimPolicy: Delete
+  compositeResource:
+    apiVersion: clusters.cnp.example.org/v1alpha1
+    kind: EKS
+  composedResources:
+  - base:
+      apiVersion: network.aws.crossplane.io/v1alpha3
+      kind: VPC
+      spec:
+        cidrBlock: 192.168.0.0/16
+        enableDnsSupport: true
+        enableDnsHostNames: true
+        providerRef:
+          name: cnp-aws
```

Comment:
@negz do we want to rely on a well known `cnp-aws` `Provider` being created out of band or instead rely on a default `Provider` as suggested in https://github.com/crossplane/example-cnp/issues/3#issuecomment-672339447?

Reply:
My preference is to add support for default providers.